### DISCUSSION TITLE
Is there a way to make the tip of the arrows not too large?

### DISCUSSION THREAD

**Discussion by vladQll** [xournalpp/xournalpp#5324] [Q&A]
My monitor is 24 inches, I use the application in full screen, and I check the fit screen option which puts it at 287% zoom, in short I can write normally with my tablet but the arrowheads look very big, and sometimes when I work with geometric constructions, they visually take up a lot of space. I am not really sure if it is my configuration that causes this but I wanted to know if there is a way to make the arrowheads to a smaller one. 
xournal:
![image](https://github.com/xournalpp/xournalpp/assets/28522011/5984d1fc-0e10-4ceb-9f90-28036e94d24c)
for example these are the arrows of an online whiteboard
![image](https://github.com/xournalpp/xournalpp/assets/28522011/f489a81c-dcf0-45c9-9171-c93992a65ada)

**Comment by tattsan** [comment]
1. Check `Preserve line width` in `Preferences -> Drawing Areas`. (In my case, this is always set.)
2. Draw a long arrow.
3. Select the arrow and resize it.

**Reply by vladQll** [reply]
it was not the way I wanted it to work, but maybe for the moment it will work, since it is not very intuitive to be resizing each arrow.

**Reply by tattsan** [reply]
Configurable arrow head like  https://github.com/xournalpp/xournalpp/pull/4055#issuecomment-1140418565 is not  implemented yet. So what we can do now is the workaround mentioned above. Improvements such as reducing the size of the arrowhead when the arrow is short have been realized in this #4055, and it is already merged.

**Reply by vladQll** [reply]
Merged means that I can download the program and install it and be able to use the new change, or does it mean not yet?

**Reply by tattsan** [reply]
That feature has already been included since release 1.2.0.
